src/Microsoft.DotNet.Watcher.Tools/DotNetWatcher.cs

```diff
@@ -45,6 +46,10 @@ public DotNetWatcher(ILogger logger)
                     var fileSetTask = fileSetWatcher.GetChangedFileAsync(combinedCancellationSource.Token);
                     var processTask = _processRunner.RunAsync(processSpec, combinedCancellationSource.Token);
 
+                    _logger.LogInformation("Running {execName} with the following arguments: {args}",
+                        processSpec.ShortDisplayName(),
+                        ArgumentEscaper.EscapeAndConcatenate(processSpec.Arguments));
```

Comment:
What's the difference between doing this an interpolating args in the string itself (`$""`)?